calico-cloud_versioned_docs/version-19-1/image-assurance/creating-jira-issues-for-scan-results.mdx

```diff
@@ -0,0 +1,41 @@
+---
+description: Creating Jira Issues for you scan results.
+---
+
+# Creating Jira issues for vulnerable images
+
+After your images have been found, the Manager UI can help facilitate remediation of your vulnerable through a Jira integration.
+
+This integration allows you to create Jira Issues directly from the Scan Results page and attaches information about the
+vulnerable package found.
+
+# Configuring the Jira Integration
+
+1. In the top right user icon in the Manager UI, click **Settings**.
+![jira-settings-link](/img/calico-cloud/ia-jira-show-settings-link.png)
+1. On the Settings page, click **Jira** tab.
+![ia-jira-credentials-tab-blank](/img/calico-cloud/ia-jira-credentials-tab-blank.png)
+1. Fill in the Username, Url, and API Token fields for the Jira account you want to integrate with.
+![ia-jira-credentials-page-filled-in](/img/calico-cloud/ia-jira-credentials-page-filled-in.png)
+
+Once the credentials are filled in, you can select the Jira Project to create Image Assurance Scan Result issues in.
+
+# Using the Jira Integration
+
+### Creating a new Jira issue for a Scan Result
+1. Click on the **Image Assurance** link in the nav bar to the left.
+1. Click on **All Scanned Images**.
+1. Click on the Scan Result you wish to create a Jira Issue for.
+1. Click on the **Add Ticket** button at the bottom of the Scan Result information in the top right.
+![ia-jira-add-ticket-link](/img/calico-cloud/ia-jira-add-ticket-link.png)
+1. Fill out the modal fields or leave the defaults as is.
+![ia-jira-modal](/img/calico-cloud/ia-jira-modal.png)
+1. Click **Create Jira Issue**
+
+The **Add Ticket** button will not be replaced with a link to the created Jira Issue.
+![ia-jira-after-creation](/img/calico-cloud/ia-jira-after-creation.png)
+
+The Jira issue created will include an attached CSV with all the discovered CVEs for the image.
+
+To remove the Jira Issue from the Scan Result, click on the X beside the ticket link.
```

Comment:
May be worth reiterating that this won't delete the ticket from Jira itself.

Reply:
I noticed that too and asked Ronan to change the UI text.